projects/js-packages/test-init.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
This needs to go in `projects/js-packages/connection` too, or else find somewhere for it near `js-test-runner.js`.